package.json

```diff
@@ -100,7 +100,7 @@
     "react-dom": "^18.3.1",
     "ts-node": "10.9.1",
     "tsc-alias": "^1.8.8",
-    "tsx": "4.19.3",
+    "tsx": "^4.19.3",
```

Comment:
can we please remove "^" in package.json please. I know not all of these are from your change @arddluma but can you make the change and push?

Reply:
sure! on it